src/models/tshirt/tshirt_c.cpp

```diff
@@ -446,7 +446,10 @@ extern int run(NWM_soil_parameters& NWM_soil_params,
     // Rather than read in forcing or rainfall data from file (as in original) function receives directly as param
     if(yes_aorc == TRUE) {
         for (i = 0; i < num_timesteps; i++) {
-            rain_rate[i] = (double) aorc_data[i].precip_kg_per_m2;  // assumed 1000 kg/m3 density of water.  This result is mm/h;
+            // assumed 1000 kg/m3 density of water.  The AORC data result is mm/h, so also convert to m/h.
+            // TODO: test to confirm the conversion to meters works out correctly
+            rain_rate[i] = ((double) aorc_data[i].precip_kg_per_m2) / 1000;
```

Comment:
I will note that the AORC input read from the internal Forcing object and provided as the input flux is different than this value (it is pre-processed m/s and adjusted for densitity/temperature relationship in the pre-processing.)

Reply:
The Tshirt C code uses a custom struct for holding AORC data, which got copied straight from Fred's original code.  It may just be incompatible with our preprocessed AORC data format (or, rather, needing a more complex adapter object to transfer between forms).  

I don't think it matters though (at least for now), since the _Tshirt_C_Realization_ class always handles reading the forcing data and then just passes in the raw rain rates.  I have opened #198 to keep track of either verifying this works (in case it is ever used in the future) or just taking it out.